src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/HintTextTest.kt

```diff
@@ -70,6 +71,58 @@ class HintTextTest {
   private val nomisCommentService = mock<NomisCommentService>()
   private val bankHolidayService = mock<BankHolidayService>()
   private val trancheOutcomeRepository = mock<TrancheOutcomeRepository>()
+  private val sentenceDetail = SentenceDetail(
```

Comment:
minor and optional - but i prefer to keep test data like sentenceDetail in the companion object (at the bottom of the file). Thinking being that it keeps the top of the file cleaner.